pkg/io/confio/read_defs.go

```diff
@@ -7,15 +7,20 @@ package confio
 
 import (
 	"encoding/json"
+	"fmt"
 	"os"
 
 	"github.com/IBM/vpc-go-sdk/vpcv1"
 
 	configModel "github.com/np-guard/cloud-resource-collector/pkg/ibm/datamodel"
 
 	"github.com/np-guard/vpc-network-config-synthesis/pkg/ir"
+
+	"github.com/np-guard/models/pkg/ipblock"
```

Comment:
```suggestion
	"github.com/np-guard/models/pkg/ipblock"

	"github.com/np-guard/vpc-network-config-synthesis/pkg/ir"
```
3rd-party packages should appear before internal packages